tests/quantization/test_quantized_module.py

```diff
@@ -512,3 +512,49 @@ def test_quantized_module_initialization_error():
             ordered_module_output_names=None,  # This makes the combination invalid
             quant_layers_dict={"layer1": (["input1"], "QuantizedOp")},
         )
+
+
+@pytest.mark.parametrize("model_class, input_shape", [pytest.param(FC, (100, 32 * 32 * 3))])
+def test_crt_simulation_using_cp_vl(model_class, input_shape, default_configuration):
```

Comment:
the test name is more about checking that rounding makes the circuit non-crt, there's nothing about simulation in this test, so maybe you can rename `test_crt_simulation_using_cp_vl` to `test_crt_to_non_crt_using_rouding` or else